token.go

```diff
@@ -7,14 +7,15 @@ type TokenType string
 // TokenParams is the set of parameters that can be used when creating a token.
 // For more details see https://stripe.com/docs/api#create_card_token and https://stripe.com/docs/api#create_bank_account_token.
 type TokenParams struct {
-	Params
-	Card     *CardParams
-	Bank     *BankAccountParams
-	PII      *PIIParams
-	Customer string
+	Params   `form:"*"`
+	Card     *CardParams        `form:"card"`
+	Bank     *BankAccountParams `form:"bank_account"`
+	PII      *PIIParams         `form:"pii"`
+	Customer string             `form:"customer"`
+
 	// Email is an undocumented parameter used by Stripe Checkout
 	// It may be removed from the API without notice.
-	Email string
+	Email string `form:"email"`
```

Comment:
A bit random here but does this comment mean that Stripe Checkout uses our official Go library? I'm assuming they just use the Token endpoint though might be worth double checking with the team for this change

Reply:
I don't know the history here, but I just looked at the project, and it doesn't appear to be go anymore.  I didn't find any instances of imports of this library.

I added a note for myself to remove this after pulling in this PR.